docs/cn/Clients-Python.md

```diff
@@ -0,0 +1,103 @@
+布局：全局
```

Comment:
Please leave the header key as English, otherwise it will not be respected as header.
```
---
layout: global
title: Python客户端
nickname: Python客户端
group: Clients
priority: 5
---
```